src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/TidligereVedtaksperioderInfo.tsx

```diff
@@ -109,25 +57,31 @@ const TidligereVedtaksperioderInfo: FC<{ tidligereVedtaksperioder: ITidligereVed
     tidligereVedtaksperioder,
 }) => {
     return (
-        <>
-            <DivMedBottomMargin>
-                <TabellTidligereVedtaksperioder
-                    infotrygd={tidligereVedtaksperioder.infotrygd}
-                    sak={tidligereVedtaksperioder.sak}
-                />
-            </DivMedBottomMargin>
+        <Container>
+            <FlexDiv>
+                {mapIkon(VilkårInfoIkon.REGISTER)}
+
+                <Label size="small" className="tittel" as="h3">
+                    Har bruker tidligere vedtaksperioder i EF Sak eller Infotrygd
+                </Label>
+            </FlexDiv>
+            <TidligereVedtaksperiodeKort tidligereVedtaksperioder={tidligereVedtaksperioder} />
 
             <FlexDiv>
                 <Registergrunnlag />
                 <SmallTextLabel>
                     Har bruker fått stønad før desember 2008 - <span>Infotrygd (PE PP)</span>
                 </SmallTextLabel>
-                <BodyShortSmall>
+                <Tag
+                    variant={
+                        tidligereVedtaksperioder.historiskPensjon ? 'success-filled' : 'neutral'
```

Comment:
```suggestion
                        tagVariant
```
Selv liker jeg å utlede slike verdier øverst i komponenten før return statementet. Da blir komponentens innhold ofte lettere å lese. 

`const tagVariant = tidligereVedtaksperioder.historiskPensjon ? 'success-filled' : 'neutral';` kan plasseres lenger oppe altså

Reply:
Enig med deg, har gjort endringer.